modules/ckanext-ytp_main/ckanext/ytp/commands.py

```diff
@@ -464,3 +469,83 @@ def add_to_groups(ctx, config, dryrun):
     else:
         for d in data_dicts:
             get_action('group_member_create')(context, d)
+
+
+opendata_harvest_group = paster_click_group(
+    summary=u'Harvester related commands.'
+)
+
+
+@opendata_harvest_group.command(
+    u'send-status-emails',
+    help='Sends harvester status emails to configured recipients'
+)
+@click_config_option
+@click.option(u'--dryrun', is_flag=True)
+@click.pass_context
+def send_harvester_status_emails(ctx, config, dryrun):
+    load_config(config or ctx.obj['config'])
+    status = get_action('harvester_status')({}, {})
+
+    errored_runs = any(item.get('status') != 'ok' for item in status.values())
+    old_runs = any(_elapsed_since(item.get('last_run')).days > 2 for item in status.values())
```

Comment:
This will result in error if `item.get('last_run')` is None .

`_elapsed_since(None)` returns `None` and `None.days` results in AttributeError. My local environment atleast has a harvester with the `last_job_status.finished` as null.

Reply:
In my local environment it had an error so it didn't finish, so the finished time is null. The `errored_runs` will be True but will the AttributeError stop the execution?

Reply:
Okay, those were not expected situations. Adding handling.